src/components/modules/FormPage/signUpForm.tsx

```diff
@@ -63,58 +39,53 @@ export default function SignUpForm() {
   const { userData } = useUserDataContext();
   const Formik = useFormik({
     initialValues: {
-      username: "",
+      discordUsername: "",
       name: "",
       email: "",
       age: 0,
-      skills: "",
+      level: 0,
```

Comment:
level is not a number, level is three possible strings only. 
beginner, intermediate, advanced.

you can have an empty string here as an initial value tho